portfolio/src/main/webapp/index.html

```diff
@@ -178,12 +184,11 @@ <h2><b>Comment Section</b></h2>
           </select>
           <br></br>
           <label class="label">First name:</label>
-          <input type="text" name="userFirstName"><br></br>
+          <input type="text" name="userFirstName"><br><br>
```

Comment:
Nit: It seems most (if not all) of your inputs are followed by two line breaks. I guess this is for a better view with a larger gap. A cleaner way to do it is with CSS
```
input {
  margin-bottom:
}
```

Reply:
Thanks, Kazi!